_data/sites.json

```diff
@@ -8834,6 +8834,17 @@
         ]
     },
 
+    {
+        "name": "ScholarshipPoints",
```

Comment:
Should the name be Revisões/ScholarshipPoints? I don't know the service, so I can't tell the relationship between the two

Reply:
The parent company is Edvisors. I can add them to the name, but I haven't heard of Revisões.

Reply:
It was my silly autocorrect, the intention was to write Edvisors :P